mmrazor/models/architectures/dynamic_op/base.py

```diff
@@ -1,25 +1,106 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 from abc import ABC, abstractmethod
+from typing import Any, Optional, Set
 
 from torch import nn
 
-from mmrazor.models.mutables.mutable_channel import MutableChannel
+from mmrazor.models.mutables.base_mutable import BaseMutable
 
 
 class DynamicOP(ABC):
```

Comment:
suggestion:
    add an interface, called convert_from(module). It is used to convert a basic torch module to a dynamic op.